src/main/java/org/jenkinsci/plugins/displayurlapi/actions/AbstractDisplayAction.java

```diff
@@ -74,9 +69,11 @@ DisplayURLProvider lookupProvider() {
         }
         DisplayURLProvider displayURLProvider = DisplayURLProvider.getPreferredProvider();
         if (displayURLProvider == null) {
-            Iterable<DisplayURLProvider> all = DisplayURLProvider.all();
-            Iterable<DisplayURLProvider> availableProviders = Iterables.filter(all, Predicates.not(Predicates.instanceOf(ClassicDisplayURLProvider.class)));
-            displayURLProvider = Iterables.getFirst(availableProviders, DisplayURLProvider.getDefault());
+            ExtensionList<DisplayURLProvider> all = DisplayURLProvider.all();
+            displayURLProvider = all.stream()
+                .filter(((Predicate<DisplayURLProvider>) ClassicDisplayURLProvider.class::isInstance)
```

Comment:
Java 11 will improve this with [`not(ClassicDisplayURLProvider.class::isInstance)`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/function/Predicate.html#not(java.util.function.Predicate))